msgcntr/messageforums-app/src/webapp/js/forum_movethread.js

```diff
@@ -314,7 +316,7 @@ Licenses.
     };
     
     var initDialog = function () {
-        $(".topic-picker").dialog({ autoOpen: false, width: 680, modal: true, position: 'top'});
+        $(".topic-picker").dialog({ autoOpen: false, width: 680, modal: true, position: "top", draggable: false, resizable: false});

```

Comment:
If this is an opinionated change, can you explain the reasoning?

Reply:
It's part of Bill's original work. I tested it without this change, and it has the following problems:

* the ability to move or "drag" the modal doesn't work. You get a different cursor to indicate you can move it, but clicking and dragging it doesn't do anything.
* the ability to resize the modal works, but it doesn't serve much of a purpose due to the width being set to 680:

![screenshot_20180303_115814](https://user-images.githubusercontent.com/10403943/36936972-356aa5c6-1f04-11e8-91fb-0de930d0a525.png)

Reply:
I don't see @cute-idiom in the list of signed CLAs

Reply:
@mhall can you have Bill submit a CLA?

Reply:
@ottenhoff is the ICLA form down? https://www.apereo.org/licensing/agreements/icla loads a page with just header and footer, no usable form, for me in FF and Chrome

Reply:
Working for me in Chrome right now

Reply:
Very weird -- our UVA institution network must be blocking something, because the form loads for me on my iPhone when on LTE but not when I connect it to institutional wifi. In any event, I'll get Bill to submit the form, thanks.

Reply:
@bjones86 Bill's CLA has been submitted